.gitignore

```diff
@@ -101,6 +101,9 @@ asv_bench/pandas/
 # Documentation generated files #
 #################################
 doc/source/generated
+doc/source/api/
+doc/source/user_guide/generated/
```

Comment:
those two should not be needed (I fixed the last one) 
They may still be there from previous version, but you should be able to remove the directories